seaborn_objects_recipes/recipes/lowess.py

```diff
@@ -5,23 +5,48 @@
 from seaborn._stats.base import Stat
 import statsmodels.api as sm
 
-
 @dataclass
 class Lowess(Stat):
-    """
-    Fit a locally-weighted regression to smooth the data.
-    """
-
-    frac: float = 0.2  # Fraction of data to use when estimating each y-value
-    gridsize: int = 100  # How fine-grained to plot the curve. Increase if jagged.
+    frac: float = 0.2
+    gridsize: int = 100
+    num_bootstrap: int = 200
+    confidence_level: float = 0.95
```

Comment:
I could default to `False` for this and `num_bootstrap`, and use the same keywords you use for regression (`alpha`). 

Bootstrapping is slow, so we usually want default behaviors to be the fast one. 

Also do some type checking on passed arguments and raise exceptions if they pass something invalid. Again, a thing we don't do in class but is good practice in software engineering public libraries (always "fail gracefully" — i.e. give an error with an informative error message).